srcts/src/shiny/bind.ts

```diff
@@ -320,6 +320,10 @@ async function bindOutputs(
       $el.addClass("shiny-bound-output");
       if (!$el.attr("aria-live")) $el.attr("aria-live", "polite");
 
+      if (Shiny.shinyapp?.$outputProgress.isRecalculating(id)) {
+        bindingAdapter.showProgress(true);
+      }
```

Comment:
@jcheng5 can you think of a scenario where the progress state would be anything other than `"initial"` here? If not, maybe we `showProgress(true)` unconditionally?

Reply:
Yes, it could be "value", "error", "cancel", or even potentially "persisting" as well. In that example app, if you toggle the Show checkbox without resampling.